### DISCUSSION TITLE
Use with Mopidy

### DISCUSSION THREAD

**Discussion by koying** [music-assistant/support#439] [Show and tell]
In order to use Music Assistant with Mopidy, through the integration available in HACS (https://github.com/bushvin/hass-integrations) you have to tweak the `mopidy.conf` as follows:

- open `mopidy.conf`in your editor of choice. Depending on your configuration, that might be `/etc/mopidy/mopidy.conf` or `~/.config/mopidy/mopidy.conf`
- Edit the `stream` section as below.  The key is the `metadata_blacklist` value

```
[stream]
enabled = true
protocols =
    http
    https
    mms
    rtmp
    rtmps
    rtsp
timeout = 5000
metadata_blacklist = http://*:8095/*
```

This is needed or Mopidy will first try to retrieve metadata from the stream provided, confusing Music Assistant that will close the stream immediately thereafter.

**Comment by ebiscaia** [comment]
Would you mind explaining how you made the integration work with Music Assistant? I mean which music provider you chose.

Thanks
